nengo_viz/components/pointer.py

```diff
@@ -1,26 +1,27 @@
 import nengo
 import nengo.spa
+from nengo.spa.module import Module
 import numpy as np
-import struct
 
 from nengo_viz.components.component import Component, Template
 
 
 class Pointer(Component):
-    def __init__(self, viz, config, uid, obj):
+    def __init__(self, viz, config, uid, obj, **args):
```

Comment:
Hmm...  this doesn't quite work as it is, since the args don't get saved in the config.  I think this is more a bug in the Template system, though (I think the correct fix is a change to the ```Template.code_python``` function).  So this is probably fine for now.